cli/board/details.go

```diff
@@ -47,42 +48,54 @@ func runDetailsCommand(cmd *cobra.Command, args []string) {
 	})
 
 	if err != nil {
-		formatter.PrintError(err, "Error getting board details")
+		feedback.Errorf("Error getting board details: %v", err)
 		os.Exit(errorcodes.ErrGeneric)
 	}
-	if output.JSONOrElse(res) {
+
+	if globals.OutputFormat == "json" {
+		feedback.PrintJSON(res)
+	} else {
 		outputDetailsResp(res)
 	}
 }
 
 func outputDetailsResp(details *rpc.BoardDetailsResp) {
-	table := output.NewTable()
-	table.SetColumnWidthMode(1, output.Average)
-	table.AddRow("Board name:", details.Name)
+	// Table is 5 columns wide:
+	// |               |                             | |                       |
+	// Board name:     Arduino Nano
+	//
+	// Required tools: arduino:avr-gcc                 5.4.0-atmel3.6.1-arduino2
+	//                 arduino:avrdude                 6.3.0-arduino14
+	//                 arduino:arduinoOTA              1.2.1
+	//
+	// Option:         Processor                       cpu
+	//                 ATmega328P                    * cpu=atmega328
+	//                 ATmega328P (Old Bootloader)     cpu=atmega328old
+	//                 ATmega168                       cpu=atmega168
+	w := tabwriter.NewWriter(feedback.OutputWriter(), 0, 0, 2, ' ', 0)
+	table := tabby.NewCustom(w)
```

Comment:
Two critical aspects I had to fight with the many tables renderes I tried before deciding to roll out my own:

- is colored output supported?
- how does it render very big tables (or tables that may have just a single very long cell among the other)?

a good test is the output of this command:
`./arduino-cli board details esp8266:esp8266:generic`

after installing the core here:
`http://arduino.esp8266.com/stable/package_esp8266com_index.json`

Reply:
no bueno, bringing back the table...

Reply:
`table` is back, I've moved it in a root package in case we wanted to re-use it in other projects. I've also simplified a bit the layout and the color handling. For the latter, the tradeoff is having the public API depending on `color` explicitly vs. having to redefine ourselves all the colors, my proposal is to go with the former (specially because atm we only use green).

Reply:
sounds good! (for both, moving to a separate repo, and depending on `color`)